### DISCUSSION TITLE
What do I need to install to build?

### DISCUSSION THREAD

**Discussion by Kivypls** [termux/termux-packages#21799] [Q&A] [answered]
What do I need to install in Termux in order to convert from code written in pygame to an apk file?

**Comment by Biswa96** [comment]
You should ask the question to pygame developers.

**Reply by Kivypls** [reply]
> Вы должны задать этот вопрос разработчикам pygame.

I don't need the pygame library, I'm interested in how to build the apk from any library (basic installations)

**Reply by Biswa96** [reply, marked as answer]
There is already a ton of information out there in the Internet https://duckduckgo.com/?t=ffab&q=build+apk+from+pygame&ia=web

**Comment by AyhemKouki** [comment]
To convert a Pygame project into an APK file using Termux, you can use a tool called Buildozer, which automates the process of packaging Python projects for Android
